### ISSUE TITLE
Paged Search with Size limit error, return no entries

### ISSUE THREAD

**Issue by IbukunOla** [cannatag/ldap3#700]
If you do a Paged search using the api given from the extend standard namespace, you get no entries if the Size Limit Exceeded Operation Result Exception is raised and you have `raiseExpections = True` and `generator = False`.
The strategy is ASYNC if that contributes anything to this,  but i also ran the code in SYNC but had the same problem.
For testing, I had 16 entries I was attempting to pull and put a size limit of 10 to exceed the Size limit.

Here is a snippet of my code:
```
try:
  entries = self.conn.extend.standard.paged_search(search_base_dn, search_filterstr,
                               attributes=search_attrlist, size_limit=10,
                               paged_size=3, search_scope='SUBTREE', generator=False)
            #Note: Size Limit Error already handles in this api
except ldap_excp.LDAPOperationResult, e:
  print(str(e))
  print(entries)
  print(self.conn.response)
```

**Comment by cannatag** [comment]
Thanks, can you try the code in dev? It should work now.

**Comment by IbukunOla** [comment]
It works in dev
I never get the Size Limit Error too. 
For example, if page size is 3 and size limit is 1, I don't get the Size Limit Error, which I think is how it is suppose to be, but I'm not 100% on that.

**Comment by cannatag** [comment]
released in v2.6.1
